csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T].cs

```diff
@@ -6,22 +6,19 @@
 namespace Platform.Collections.Segments.Walkers
 {
     /// <summary>
-    /// <para>All Segment sWalker Base or Class with WalkAll method to walk through all segments in the dictionary.</para>
-    /// <para>Базовый проходчик всех сегментов или Класс с методом WalkAll для прохода по всем сегментам в словаре.</para>
+    /// <para>Provides the base class from which the classes that represent all segments walker are derived. This is an abstract class.</para>
+    /// <para>Предоставляет базовый класс, от которого наследуются классы, реализующие AllSegmentsWalkerBase. Это абстрактный класс.</para>
     /// </summary>
     /// <seealso cref="AllSegmentsWalkerBase{T, Segment{T}}"/>
     public abstract class AllSegmentsWalkerBase<T> : AllSegmentsWalkerBase<T, Segment<T>>
     {
-         /// <summary>
-        /// <para>Calls the segment constructor, initializes 3 fields on the segment instance</para>
-        /// <para>Вызывает конструктор сегмента, инициализирует 3 поля у экземпляра сегмента.</para>
+        /// <summary>
+        /// <para>Initializes a new instance of the <see cref="Segment"/> class, using the <paramref name="base"/> list of the segment, Offset relative to the list<paramref name="offset"/> and The segment's length <paramref name="length" />.</para>
```

Comment:
Things like "the base parameter is the a of the segment", "the offset parameter is an offset relative to the list", "the length parameter is an length of the segment" specified in param tags, so we should follow the programming rule [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). 
`Initializes a new instance of the <see cref="Segment"/> class, using the <paramref name="base"/>, <paramref name="offset"/>, <paramref name="length"/>.`